packages/mqueue/mqueue.0.5.2-0/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+maintainer: "Markus W. Weissmann <markus.weissmann@in.tum.de>"
+authors: "Markus W. Weissmann <markus.weissmann@in.tum.de>"
+homepage: "http://mqueue.forge.ocamlcore.org/"
+bug-reports: "http://mqueue.forge.ocamlcore.org/"
+dev-repo: "git://mqueue.forge.ocamlcore.org/"
+license: "MIT"
+build: [
+  ["ocaml" "setup.ml" "-configure"]
```

Comment:
I think you need to add ` "--prefix" prefix`.  [oasis2opam](https://github.com/ocaml/oasis2opam) can automate this.